File: src/classes/api/req_res/get-gkill-notification-public-key-response.ts
'use strict'

import { GkillAPIResponse } from '../gkill-api-response'

export class GetGkillNotificationPublicKeyResponse extends GkillAPIResponse {

    gkill_notification_public_key: string

    constructor() {
        super()
        this.gkill_notification_public_key = ""
    }

}

File: src/app/gkill/api/req_res/get_gkill_notification_public_key_response.go
package req_res

import "github.com/mt3hr/gkill/src/app/gkill/api/message"

type GetGkillNotificationPublicKeyResponse struct {
	Messages []*message.GkillMessage `json:"messages"`

	Errors []*message.GkillError `json:"errors"`

	GkillNotificationPublicKey string `json:"gkill_notification_public_key"`
}
